internal/moduletest/hcl/provider.go

```diff
@@ -27,7 +27,7 @@ type ProviderConfig struct {
 	Original hcl.Body
 
 	ConfigVariables     map[string]*configs.Variable
```

Comment:
Can we delete this field now? I think we don't need it if we're parsing the variables at the higher level every time?

Reply:
Great catch, I woudn't have noticed!